File: Easy/1437 - atleastlengthkplacesaway/atleastlengthkplacesaway.py
class Solution:
    def kLengthApart(self, nums: list[int], k: int) -> bool:
        j = k

        for i in range(len(nums)) :
            if (nums[i] == 1) :
                if (j < k) :
                    return False
                j = 0
            else :
                j += 1
        return True

File: Easy/1437 - atleastlengthkplacesaway/atleastlengthkplacesaway.js
var kLengthApart = function(nums, k) {
    let j = k;

    for (let i = 0; i < nums.length; i++)
    {
        if (nums[i] == 1)
        {
            if (j < k)
                return false;
            j = 0;
        }
        else
            j++;
    }
    return true;
};